src/test/java/com/google/sps/data/RecommendationEngineTest.java

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import java.util.*;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class RecommendationEngineTest {
+  private Drink DRINKA = new Drink("latte", "123", 4.8, 12, null);
+  private Drink DRINKB = new Drink("cold brew", "123", 4.8, 12, null);
+  private Drink DRINKC = new Drink("latte", "456", 4.2, 12, null);
+  private Drink DRINKD = new Drink("cold brew", "456", 4.2, 12, null);
+  private Drink DRINKE = new Drink("latte", "789", 1.5, 12, null);
+  private Drink DRINKF = new Drink("cold brew", "789", 1.5, 12, null);
+  private List<Drink> drinkList = new ArrayList<>();
+  private static final double DELTA = .0000001;
+
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  @Test
+  public void testGetBestShop() {
```

Comment:
Suggestion (can be handled in a subsequent pull request given time considerations):
A good practice is for the test name to include:
a) the condition being tested, and
b) the expected result.

Examples:
- testGetDrinkScore_returnsCorrectResult
- transferMoney_addsToDestinationAccount
- testGetBestShop_storeNotFound

Reply:
i understand, i will incorporate this in my next pr

Reply:
Let's create an issue for this and other TODOs to keep track of pending work.